lib/rules/yoda.js

```diff
@@ -54,13 +54,15 @@ function looksLikeLiteral(node) {
 /**
  * Attempts to derive a Literal node from nodes that are treated like literals.
  * @param {ASTNode} node Node to normalize.
+ * @param {number} [defaultValue] The default value to be returned if the node
+ *                                is not a Literal.
  * @returns {ASTNode} The original node if the node is already a Literal, or a
  *                    normalized Literal node with the negative number as the
  *                    value if the node represents a negative number literal,
  *                    otherwise null if the node cannot be converted to a
  *                    normalized literal.
```

Comment:
Can you update the return value description to note when it will return the default value?